apps/src/lab2/views/components/Instructions/ResourcePanel/version-history-panel.module.scss

```diff
@@ -21,20 +21,88 @@ $space-xl: 10px;
   overflow-y: auto;
   flex: 1;
   min-height: 0;
+  position: relative;
+
+  /* Simple background line that runs underneath the radio buttons */
+  background: linear-gradient(to right, transparent 32px, var(--borders-neutral-primary) 32px, var(--borders-neutral-primary) 33px, transparent 33px);
+}
+
+.rowContainer {
+  display: flex;
+  padding: var(--padding-p-xxs, 8px) 12px;
+  align-items: flex-start;
+  gap: 10px;
+  align-self: stretch;
+  border-radius: var(--border-radius-br-s, 4px);
+  border: 1px solid var(--borders-neutral-primary);
+  background: var(--background-neutral-secondary);
+  margin-left: 8px;
+  margin-right: 8px;
+  position: relative;
+  z-index: 1;
 }
 
-.row {
-  padding: $space-l;
+.isRow {
```

Comment:
nit: I would name these something like `row`, `commentRow`, etc. Is there a reason for putting `is` in front?